hazelcast/proxy/set.py

```diff
@@ -54,6 +63,7 @@ def add_all(self, items):
         return self._invoke(request, set_add_all_codec.decode_response)
 
     def add_listener(self, include_value=False, item_added_func=None, item_removed_func=None):
+        # type: (bool, Callable[[ItemEvent],None], Callable[[ItemEvent],None]) -> Future[str]
```

Comment:
It would be good to leave a space between `[ItemEvent],None` like you did in the other file to make them consistent (please apply this to other places too)

Reply:
Yeah, you are right. I usually take care it, but I missed this.